src/main/java/com/hedvig/backoffice/graphql/resolvers/ClaimResolver.kt

```diff
@@ -149,13 +152,19 @@ class ClaimResolver(
         }
     }
 
-    fun getContract(claim: Claim): Contract? {
-        return if (claim.contractId != null) {
-            productPricingService.getContractById(claim.contractId)
-        } else null
-    }
+    fun getContract(claim: Claim): Contract? = if (claim.contractId != null) {
+        productPricingService.getContractById(claim.contractId)
+    } else null
```

Comment:
How do you feel about:

```suggestion
    fun getContract(claim: Claim): Contract? = claim.contractId?.let { id ->
        productPricingService.getContractById(id)
    }
```
?

I always find the trailing `else null` to look kind of awkward ^^